pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -197,14 +216,22 @@ stdenv.mkDerivation rec {
   cmakeFlags = [
     "-DINSTALL_PLUGINSDIR=${qtPluginPrefix}"
     "-DINSTALL_QMLDIR=${qtQmlPrefix}"
-    "-DQT_FEATURE_journald=ON"
-    "-DQT_FEATURE_sctp=ON"
     "-DQT_FEATURE_libproxy=ON"
     "-DQT_FEATURE_system_sqlite=ON"
-    "-DQT_FEATURE_vulkan=ON"
     "-DQT_FEATURE_openssl_linked=ON"
+  ] ++ lib.optionals (!stdenv.isDarwin) [
+    "-DQT_FEATURE_sctp=ON"
+    "-DQT_FEATURE_journald=ON"
+    "-DQT_FEATURE_vulkan=ON"
+  ] ++ lib.optionals stdenv.isDarwin [
+    "-DFEATURE_framework=OFF"
```

Comment:
What is this and why do we need to turn it off?

Reply:
This disables building Apple frameworks.

Reply:
And why don't we want that?

Reply:
We would definitely prefer Qt to be built [as a set of dylibs](https://doc.qt.io/qt-6/macos-building.html) to fit into nixpkgs.

Reply:
Could you add that as a comment?

Reply:
Done.